From 2c3d0f5a8626a2e932eb2e5546a8c942aeab16a7 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 14 Sep 2016 13:15:15 -0700
Subject: [PATCH] CMON-9 add cmon builds to purge-mg-builds process Reviewed
 by: Richard Kiene <richard.kiene@joyent.com> Approved by: Richard Kiene
 <richard.kiene@joyent.com>

---
 tools/purge-mg-builds | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/purge-mg-builds b/tools/purge-mg-builds
index 964ec17..0271476 100755
--- a/tools/purge-mg-builds
+++ b/tools/purge-mg-builds
@@ -97,6 +97,8 @@ TTL_DAYS_FROM_NAME='{
     "binder": 365,
     "ca": 365,
     "cloudapi": 365,
+    "cmon": 365,
+    "cmon-agent": 365,
     "cn-agent": 365,
     "cnapi": 365,
     "cns": 365,
-- 
2.21.0

